functions/lib/integration/import-product.js

```diff
@@ -152,9 +146,11 @@ module.exports = ({ appSdk, storeId, auth }, tinyToken, queueEntry, appData, can
                 if (productId) {
                   method = 'PATCH'
                   endpoint = `/products/${productId}.json`
-                } else {
+                } else if (!productId && (tipo !== 'estoque' || !tipo)) {
                   method = 'POST'
                   endpoint = '/products.json'
+                } else {
+                  return
```

Comment:
```suggestion
                  return null
```

void não, o que não termina em request resolve com null https://github.com/ecomplus/app-tiny-erp/blob/4362d80626b4e46fcd9803fe75b4e320ae684cf9/functions/lib/integration/import-product.js#L113